docs/decisions/0014-no-new-apps.rst

```diff
@@ -0,0 +1,32 @@
+14. Documentation of new apps in edx-platform
+
+Status
+------
+In Review
+
+Context
+-------
+In recent years, edX and the software industry as a whole have been moving away from a monolith structure towards more modular architecture. The goal is to transition from a base site with plugins to a collection of more encapsulated services that can be used as building blocks to a larger application. To that end, much of the original edx-platform repository has been split out into micro-frontends, other microservices, plugins, and libraries. However, there are still a number of optional applications within edx-platform and new ones continue to be added.
```

Comment:
Thinking about whether the first sentence could be tightened a bit, and the second leverage some language from our arch docs/presentations by saying something like: 

"The Open edX platform is moving toward a more modular architecture.  The goal is to transition from a monolithic application in edx-platform to one in which this repository represents a small, stable core with volatility pushed into extensions and plugins.  To that end, ..."